ports/qemu-riscv/modmachine.c

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Looks like modmachine broke in the most recent rebase, due to refactorings. I pushed a fix here:
https://github.com/projectgus/micropython/commit/aecbf29df604f1e7e76ca68ede7902942c3c71f9

Reply:
Thanks, I'll take those changes in!